polyply/src/build_file_parser.py

```diff
@@ -96,6 +104,11 @@ def _distance_restraints(self, line, lineno=0):
             tol = 0.0
 
         for idx in self.current_molidxs:
+            if nodes[0] not in self.topology.molecules[idx]:
+                raise IOError("Could not find atom {node} in molecule with index {idx}.".format(node=nodes[0], idx=idx))
+            elif nodes[1] not in self.topology.molecules[idx]:
+                raise IOError("Could not find atom {node} in molecule with index {idx}.".format(node=nodes[1], idx=idx))
```

Comment:
Does it add something to mention the molecule name? Would that be available to begin with?

Reply:
I guess I was just lazy